FirstTask.py

```diff
@@ -0,0 +1,230 @@
+            pass_code = int(input("Enter Passcode "))
+            while(1):
+                ACCOUNT_TYPES = ["Current","Saving","Business"]
+                account_select = int(input("Select Account Type from following:\n1) Current \n2) Saving \n3) Business\n"))
+                if account_select == 1:
+                    account_type = ACCOUNT_TYPES[1]
+                    break
+                elif account_select == 2:
+                    account_type = ACCOUNT_TYPES[2]
+                    
+                    break
+                elif account_select == 3:
+                    account_type = ACCOUNT_TYPES[3]
+                    break
+                else:
+                    print("\nInvalid Account Type\n")
+            bank.create_account(name, pass_code,account_type)
+            print(f"Account Created Successfully\nYour Account Number is {account_number_id}\n\n")
+            input("\nPress Any Key to Continue\n")
+            sys("clear")
+        
+        elif option == 2:
+            if bank.is_logged_in() == False:
+                account_number = int(input("Enter Account Number: "))
+                code = int(input("Enter Passcode: "))
+                log = bank.login(account_number, code)
+
+            if bank.is_logged_in:
+                print("Login Successful!")
+                input("\nPress Any Key to Continue\n")
+                select = 1
+                while not select == 0:
+                    sys('clear')
+                    select = int(input("1) Check Balance \n2) Deposit Bill \n3) Draw Money \n4) Transfer Money \n5) Account Type \n6) Compare Your Account \n7) Account Details \n0) Log Out \nChoose:"))
+                    if select == 1:
+                        print(log[1].check_balance())
+                        input("\nPress Any Key to Continue\n")
+
+                    elif select == 2:
+                        deposit_money = int(input("Enter Deposit Amount: "))
+                        if log[1].deposit_bill(deposit_money):
+                            print("Money Successfully Deposited")
+                        else:
+                            print("Money must be Greater than 0!")
+                        input("\nPress Any Key to Continue\n")
+
+                    elif select == 3:
+                        amount = int(input("Enter Amount to withdraw: "))
+                        if log[1].draw_money(amount):
+                            print("Money Draw Successfull")
+                        else:
+                            print("Not Enough Cash")
+                        input("\nPress Any Key to Continue\n")
+                    
+                    elif select == 4:
+                        money = int(input("Enter Transfer Amount: "))
+                        receiver_account_number = int(input("Enter Account Number of Other User: "))
+                        value = bank.transfer_money(log[1].account_number(), receiver_account_number, money)

```

Comment:
Now `value` should be `status, response` where `response would always be a message which can be printed here.